src/components/AddComment.jsx

```diff
@@ -0,0 +1,72 @@
+import { postComment } from "../utils/Api";
+import React, { useState, useContext } from "react";
+import { useParams } from "react-router";
+
+import { UserContext } from "../utils/Context";
+
+const AddComment = ({ comments, setComments }) => {
+  const [comment, setComment] = useState("");
+  const { user } = useContext(UserContext);
+  const { article_id } = useParams();
+  console.log(user);
```

Comment:
Make sure to remove console logs you don't need

Reply:
Done